manifests/cf-manifest/operations.d/229-cf-update_cflinuxfs_releases.yml

```diff
@@ -3,6 +3,6 @@
   path: /releases/name=cflinuxfs3
   value:
     name: "cflinuxfs3"
-    version: "0.352.0"
-    url: "https://bosh.io/d/github.com/cloudfoundry/cflinuxfs3-release?v=0.352.0"
-    sha1: "7e33fc1207aed7616e76cdc2fc3f68919ebe256a"
+    version: "0.361.0"
```

Comment:
this was already bumped to 361 in https://github.com/alphagov/paas-cf/pull/3174 and it's also being bumped to 362 in https://github.com/alphagov/paas-cf/pull/3178 
suggest we remove this change and rebase from master to include 361

Reply:
This was added *because* I rebase from master, which is odd...however not a conflict so it'll be okay.